grafana/scylla-detailed.master.template.json

```diff
@@ -465,10 +465,19 @@
                         "title": "Writes currently blocked on commitlog"
                     },
                     {
-                        "class": "text_panel",
-                        "content": "",
-                        "mode": "markdown",
-                        "span": 3
+                        "class": "graph_panel",
+                        "span": 3,
+                        "targets": [
+                            {
+                                "expr": "clamp_max(1 + sum(rate(scylla_cache_row_hits[1m])) by ([[by]])/(sum(rate(scylla_cache_row_misses[1m])) by ([[by]]) + 0.00001),100)",
```

Comment:
What's the purpose of this + 0.00001?

Reply:
Staty away from division by zero

Reply:
Oh, I couldn't follow all these parentheses...